container-registry/container-analysis/src/test/java/com/example/containeranalysis/SamplesTest.java

```diff
@@ -0,0 +1,256 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.containeranalysis;
+
+import static java.lang.Thread.sleep;
+import static junit.framework.Assert.fail;
+import static junit.framework.TestCase.assertEquals;
+
+import com.google.cloud.devtools.containeranalysis.v1beta1.GrafeasV1Beta1Client;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.cloud.pubsub.v1.SubscriptionAdminClient;
+import com.google.pubsub.v1.ProjectSubscriptionName;
+import io.grafeas.v1beta1.Note;
+import io.grafeas.v1beta1.Occurrence;
+import io.grafeas.v1beta1.vulnerability.Details;
+import io.grafeas.v1beta1.vulnerability.Vulnerability;
+import io.grpc.StatusRuntimeException;
+import java.util.Date;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * test runner
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class SamplesTest {
+
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String subId = "CA-Occurrences-" + (new Date()).getTime();
+  private static GrafeasV1Beta1Client client;
+  private String noteId;
+  private String imageUrl;
+  private Note noteObj;
+  private static final int SLEEP_TIME = 1000;
+  private static final int TRY_LIMIT = 10;
+
+  @Rule
+  public TestName name = new TestName();
+
+  @BeforeClass
+  public static void setUpClass() throws Exception {
+    client = GrafeasV1Beta1Client.create();
+  } 
+
+  @AfterClass
+  public static void tearDownClass() {
+    try {
+      client.shutdownNow();
+      SubscriptionAdminClient subscriptionAdminClient = SubscriptionAdminClient.create();
+      ProjectSubscriptionName subName = ProjectSubscriptionName.of(PROJECT_ID, subId);
+      subscriptionAdminClient.deleteSubscription(subName);
+      subscriptionAdminClient.shutdownNow();
+    } catch (Exception e) {
+      // these exceptions aren't relevant to the tests
+    }
+  }
+
+
+  @Before
+  public void setUp() throws Exception {
+    System.out.println(name.getMethodName());
+    noteId =  "note-" + (new Date()).getTime() + name.getMethodName();
+    imageUrl = "www." + (new Date()).getTime() + name.getMethodName() + ".com";
+    noteObj = Samples.createNote(client, noteId, PROJECT_ID);
+  }
+
+  @After
+  public void tearDown() {
+    try {
+      Samples.deleteNote(client, noteId, PROJECT_ID);
+    } catch (Exception e) {
+      // these exceptions aren't relevant to the tests
```

Comment:
We should at least write exception out - if something does go wrong, it will be helpful to see it in the tests